src/Workspaces/Core/Portable/Classification/SyntaxClassification/ISyntaxClassifier.cs

```diff
@@ -24,12 +23,12 @@ internal interface ISyntaxClassifier
         /// This method will be called for all nodes that match the types specified by the SyntaxNodeTypes property.
         /// Implementations should return null (instead of an empty enumerable) if they have no classifications for the provided node.
```

Comment:
>/// [](start = 8, length = 3)

Not related to this PR: the comment seems out-of-date (there is no return value and no enumerable...).

Reply:
indeed :)   will fix.